precompile/contracts/nativeminter/nativemintertest/solidity/ERC20NativeMinter.sol

```diff
@@ -2,26 +2,30 @@
 pragma solidity ^0.8.24;
```

Comment:
can we get rid of this file? I don't think this is a viable testing for this and we should not be importing any extra contracts.

Reply:
Instead we can use a much simpler approach that only issues `mintNativeCoin` (similar to allowlist tests.)

Reply:
Yup I've deleted this file entirely -- let me know what you think of `NativeMinterTest`.